index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="zxx" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <title>ADOPTADOG HOME PAGE</title>
+  </head>
+  <body>
+    <header>
+      <div class="header">
+        <img class="header-img" src="images/header-image.jpg" alt="Brown dog with soulful eyes looking out a window">
+        <div class="header-logo">
+          <img class="header-logo" src="images/adoptadog-logo.svg" alt="Adoptadog logo">
+          <h1>Seattle’s premiere dog adoption agency</h1>
+        </div>
+      </div>
+      <div class="header-checkout">
+        <p><img src="images/cart-icon.svg" width="25" alt="shopping cart">
+        $ 1234.56</p>
+        <a href="checkout.html">Checkout</a>
+      </div>
+    </header>
+
+    <nav>
+      <a href="index.html">Home</a> /
+      <a href="dog.html">Dogs</a> /
+      <a href="blog.html">Blog</a>
+    </nav>
+    <div class="container">
+      <article>
+        <div class="home-container">
+          <img src="images/home-intro.jpg" width="300" alt="Golden Retriever being hugged around the neck by its owner">
+          <h2>Adopt A Dog</h2>
+          <p>Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga. Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium hillupta aut es ut alitatuscit ommossum haritatur arum qui officae videbiti corporeium faccull oribus es quidignis ipietus explam sus
+           am aut amet ant fugiatum, utem non reptat ad magnienecto volorent fugia nonsedi gendand itatquos as magnimpor aut molor autatia ad est officimus accae et omnia cus, od qui ate</p>
+        </div>
+
+        <div class="dog-profiles">
+          <section>
+            <img src="images/murphy-card.jpg" width="180" alt="Murphy">
```

Comment:
You should not have any inline styles, especially since this is interpreted by browsers as px and will overwrite any responsive css you have in your stylesheet.

Reply:
I moved it to css.